Mastodon/Scene/Profile/UserLIst/UserListViewModel+State.swift

```diff
@@ -123,40 +124,61 @@ extension UserListViewModel.State {
             
             Task {
                 do {
-                    let response: Mastodon.Response.Content<[Mastodon.Entity.Account]>
+                    let accountResponse: Mastodon.Response.Content<[Mastodon.Entity.Account]>
                     switch viewModel.kind {
                     case .favoritedBy(let status):
-                        response = try await viewModel.context.apiService.favoritedBy(
+                            accountResponse = try await viewModel.context.apiService.favoritedBy(
```

Comment:
This looks like one level of indentation too much, or maybe I'm seeing it wrong?